styles.css

```
@@ -19,15 +22,20 @@
   border: 5px;
   border-radius: 5px;
   border-color: rgb(73, 10, 51);
-  width: 50%;
+  width: 100%;
+  height: 80px;
   text-align: center;
+  text-wrap: balance;
+}
+
+.fa-10x {
+  align-content: flex-end;
```

I guess, you need to use `align-self` to align this element within a flexbox.
```suggestion
  align-self: flex-end;
```